src/main/java/org/opentripplanner/apis/gtfs/model/PlanPageInfo.java

```diff
@@ -0,0 +1,77 @@
+package org.opentripplanner.apis.gtfs.model;
+
+import graphql.relay.ConnectionCursor;
+import java.time.Duration;
+import java.util.Objects;
+
+public class PlanPageInfo {
+
+  private final ConnectionCursor startCursor;
+  private final ConnectionCursor endCursor;
+  private final boolean hasPreviousPage;
+  private final boolean hasNextPage;
+  private final Duration searchWindowUsed;
+
+  public PlanPageInfo(
+    ConnectionCursor startCursor,
+    ConnectionCursor endCursor,
+    boolean hasPreviousPage,
+    boolean hasNextPage,
+    Duration searchWindowUsed
+  ) {
+    this.startCursor = startCursor;
+    this.endCursor = endCursor;
```

Comment:
```suggestion
    this.startCursor = Objects.requireNonNull(startCursor);
    this.endCursor = Objects.requireNonNull(endCursor);
```
Prevent failing in eq/hc by failing fast in constructor.

Reply:
The cursors are nullable and we shouldn't prevent building this object if they are null. However, thanks for pointing this out as the eq/hc were not null safe, I've updated them now.

Reply:
Had a feeling these where nullable, but the code told another story - happy to catch the bug.